index.html

```diff
@@ -1680,11 +1771,8 @@ <h2>Authentication</h2>
             <p>
 Note that the [=verification method=] indicated by the
 `authentication` property of a [=controller document=] can
-only be used to [=authenticate=] the [=controller=]. To
-[=authenticate=] a different [=controller=], the entity associated with
-the value of `controller` needs to [=authenticate=] with its
-<em>own</em> [=controller document=] and associated
-`authentication` [=verification relationship=].
+only be used to [=authenticate=] interactive sessions on behalf 
+of the controller document's base identifier. 
```

Comment:
I don't think we need to say "interactive sessions", but +1 to fixing this to clarify that authentication is using the identifier in the document (it is *not* for authenticating the controller).

```suggestion
only be used to [=authenticate=] on behalf 
of the controller document's base identifier. 
```